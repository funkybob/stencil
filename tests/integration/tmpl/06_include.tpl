{% include "01_simple.tpl" name=first_name %}
