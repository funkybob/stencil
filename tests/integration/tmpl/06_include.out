Hello, Victor.
Everyone, it's Victor!

